src/openforms/submissions/metrics.py

```diff
@@ -60,3 +60,25 @@ def count_submissions(
     unit="",  # no unit so that the _ratio suffix is not added
     callbacks=[count_submissions],
 )
+
+upload_file_size = meter.create_histogram(
```

Comment:
I can't find this metric in prometheus. Here is a list of all otel_* metrics I have, so you can see there `submission.attachments_per_submission`, but not this one. Do you see this metric? 

<img width="634" height="490" alt="image" src="https://github.com/user-attachments/assets/3b974873-71e0-4ff6-8c01-e0ad4fcda47a" />

Reply:
have you defined a form with file uploads and actually uploaded documents + submitted the step?

These metrics are not updated from database queries, but during the runtime of application.